README.md

```diff
@@ -16,6 +16,10 @@ This is a minimalist version of a To-do List website.
 **Judá Teixeira**
 - GitHub: [@mrjuda](https://github.com/mrjuda "Judá Teixeira's GitHub profile")
 - LinkedIn: [@judateixeira](https://www.linkedin.com/in/judateixeira "Judá Teixeira's Linkedin profile")
+- Twitter: [@judalabs](https://twitter.com/judalabs)
+- Instagram: [@judalabs](https://instagram.com/judalabs)
+- Tumblr: [@judalabs](https://judalabs.tumblr.com)
+- Facebook: [@judalabs](https://facebook.com/judalabs)
 
```

Comment:
- [ ] Please it will be nice to add explicit steps to clone and run your app locally, as this will be of help to non technical users.

Reply:
> * [ ]  Please it will be nice to add explicit steps to clone and run your app locally, as this will be of help to non technical users.

Done, thank you :-)